purchase_landed_cost/models/purchase_order.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013 Joaquín Gutierrez
+# Copyright 2014-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    @api.multi
+    def action_open_landed_cost(self):
+        self.ensure_one()
+        line_obj = self.env['purchase.cost.distribution.line']
+        lines = line_obj.search([('purchase_id', '=', self.id)])
+        if lines:
+            mod_obj = self.env['ir.model.data']
+            model, action_id = tuple(
+                mod_obj.get_object_reference(
+                    'purchase_landed_cost',
+                    'action_purchase_cost_distribution'))
+            action = self.env[model].browse(action_id).read()[0]
```

Comment:
Why don't use direcly
```suggestion
            action = self.env.ref('purchase_landed_cost.action_purchase_cost_distribution').read()[0]
```
and remove previous code.

Reply:
Done